packages/adapters/src/adapters/Cornerstone3D/Angle.ts

```diff
@@ -94,7 +92,38 @@ class Angle extends BaseAdapter3D {
             rAngle: angle,
             trackingIdentifierTextValue: this.trackingIdentifierTextValue,
             finding,
-            findingSites: findingSites || []
+            findingSites: findingSites || [],
+            use3DSpatialCoordinates: false
+        };
+    }
+
+    public static getTID300RepresentationArgumentsSCOORD3D(tool) {
+        const { data, finding, findingSites } = tool;
+        const { cachedStats = {}, handles } = data;
+
+        // Using world coordinates for 3D points
+        const start = handles.points[0];
+        const middle = handles.points[1];
+        const end = handles.points[2];
+
+        const point1 = { x: start[0], y: start[1], z: start[2] };
+        const point2 = { x: middle[0], y: middle[1], z: middle[2] };
+        const point3 = point2;
+        const point4 = { x: end[0], y: end[1], z: end[2] };
+
+        const cachedStatsKeys = Object.keys(cachedStats)[0];
+        const { angle } = cachedStatsKeys ? cachedStats[cachedStatsKeys] : {};
+
+        return {
+            point1,
+            point2,
+            point3,
+            point4,
+            rAngle: angle,
+            trackingIdentifierTextValue: this.trackingIdentifierTextValue,
+            finding,
+            findingSites: findingSites || [],
+            use3DSpatialCoordinates: true
```

Comment:
This should only be true when the points are not coplanar with the selected image.  Otherwise we have to drop back to index coordinates for backwards compatibility.

Reply:
Never mind, you created a new function entirely.